epsagon/utils.py

```diff
@@ -137,6 +139,10 @@ def init(
     if ignored_keys:
         ignored_keys = ignored_keys.split(',')
 
+    allowed_keys = os.getenv('EPSAGON_ALLOWED_KEYS')
```

Comment:
I added a section documenting env variables in the README file, please add this one too

Reply:
added